src/main/java/com/mongodb/kafka/connect/sink/processor/id/strategy/PartialValueStrategy.java

```diff
@@ -62,9 +66,17 @@ public FieldProjector getFieldProjector() {
   @Override
   public void configure(final MongoSinkTopicConfig config) {
     FieldProjectionType valueProjectionType =
-        FieldProjectionType.valueOf(config.getString(VALUE_PROJECTION_TYPE_CONFIG).toUpperCase());
-
-    String fieldList = config.getString(VALUE_PROJECTION_LIST_CONFIG);
+        FieldProjectionType.valueOf(
+            getOverrideOrDefault(
```

Comment:
`getOverrideOrDefault` does the magic by checking for the overridden value first (eg. `document.id.strategy.partial.value.projection.type` then `value.projection.type`).  This keeps it backwards compatible but allows for specific configuration solely for the ProvidedIn[Key|Value]Strategy classes.